src/libraries/Microsoft.PowerFx.Core/Functions/DefaultResourceElement.cs

```diff
@@ -0,0 +1,113 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.PowerFx.Core.Functions
+{
+    public abstract class BaseResourceElement
+    {
+        public abstract bool IsUri { get; }
```

Comment:
> public abstract bool IsUri { get; } [](http://example.com/codeflow?start=7&length=36)

do we truly need this? 


Every BaseResourceElement is a uri

Reply:
I think I can remove it